templates/CSI/rbd/secret.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ csi_rbd_secret | default ('csi-rbd-secret') }}
+  namespace: {{ csi_rbd_secret_ns | default('default') }}
+data:
+  # Key value corresponds to a user name defined in Ceph cluster
+  # ceph auth get-key admin | base64
+  admin: {{ base64_encoded_admin_password }}
+  # Key value corresponds to a user name defined in Ceph cluster
+  # ceph auth get-key kubernetes | base64
+  kubernetes: {{ base64_encoded_user_password }}
```

Comment:
Can we comment it if we not we need to create  kubernetes user else pvc will remain in Pending

Reply:
same :)